src/module3/loaders/express.ts

```diff
@@ -19,6 +22,10 @@ export default async ({ app }: { app: express.Application }) => {
 
   app.use(express.json());
 
+  app.use(cors());
+
+  app.use(jwt({ secret: JWT_SECRET, algorithms: ['HS256']}).unless({path: ['/login']}));
```

Comment:
Error handling for 401/403 is missing as required in the home task. Can check the error handling part for express-jwt and customize. By checking the module code it seems it is limited to throw only Unauthorized ones.